esp/src/src/ESPUtil.ts

```diff
@@ -374,7 +374,7 @@ export const FormHelper = declare(null, {
         var t = registry.byId(this.id + timeField).attr("value");
         if (d) {
             if (t) {
-                d.setHours(t.getHours());
+                d.setHours(t.getHours() - d.getTimezoneOffset() / 60);
```

Comment:
Have you checked if this is also correct for:
* DFU Workunits
* Disk Usage

(I suspect it is and I suspect I always happened to test in UTC timezone...)

Reply:
Yes, it works in both. HOWEVER, Disk Usage Widget does not account for time although we offer that option, that code may need to be added, or those dropdowns removed in a new JIRA if we would like that to filter by time as well.